<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editar_actor</class>
 <widget class="QMainWindow" name="editar_actor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>crear_actor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imagen_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/0.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nombre_in">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>70</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_imagen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cambiar imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del actor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de nacimiento:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="nacimiento_in">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>7</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2015</year>
      <month>8</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>7915</year>
      <month>8</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>1753</year>
      <month>9</month>
      <day>14</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Genero:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_guardar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_limpiar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cancelar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="genero">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Hombre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mujer</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="actor_id">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:XXXXXXXXXX</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
